MicaForEveryone.App/Strings/hu-HU/Resources.resw

```diff
@@ -0,0 +1,291 @@
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AboutCard.Header" xml:space="preserve">
+    <value>Mica For Everyone névjegye</value>
+  </data>
+  <data name="AboutLabel.Text" xml:space="preserve">
+    <value>Névjegy</value>
+  </data>
+  <data name="AppSettingsPage.Header" xml:space="preserve">
+    <value>App beállítások</value>
+  </data>
+  <data name="AppTitleTextBlock.Text" xml:space="preserve">
+    <value>Mica For Everyone</value>
+  </data>
+  <data name="BackdropTypeCard.Header" xml:space="preserve">
+    <value>Háttér típus</value>
+  </data>
+  <data name="BlurBehindCard.Header" xml:space="preserve">
+    <value>Elmosott háttér engedélyezése</value>
+  </data>
+  <data name="CheckForUpdatesButton.Content" xml:space="preserve">
+    <value>Frissítések keresése</value>
+  </data>
+  <data name="CheckForUpdatesCard.Header" xml:space="preserve">
+    <value>Frissítések keresése</value>
+  </data>
+  <data name="CornerPreferenceCard.Header" xml:space="preserve">
+    <value>Frissítések keresése</value>
```

Comment:
This seems to be translated wrong. It matched the other two "check for updates" strings and when I used Bing Translate it confirmed that this meant "Check for updates"